modules/apps/account/account-service/src/main/java/com/liferay/account/model/impl/AccountEntryImpl.java

```diff
@@ -32,6 +41,20 @@
  */
 public class AccountEntryImpl extends AccountEntryBaseImpl {
 
+	@Override
+	public Contact fetchContact() throws PortalException {
+		List<Contact> contacts = ContactServiceUtil.getContacts(
```

Comment:
why are you wrapping the getContacts method in a method named fetch? it looks wrong to me

Reply:
as far as I know the `get` should throw an exception if not found, instead the `fetch` could be null (but I can be wrong)

Reply:
because I'm returning null if not found, not throwing an exception

Reply:
Now I got it, there is no method fetch in ContactServiceUtil, there is only the getContacts, that should be named fetch